bread/layout/base.py

```diff
@@ -66,53 +77,28 @@ def resolve(self, context):
         return str(format_value(super().resolve(context)))
 
 
-# notes on changes (this may be removed py wipascal):
-# By inheriting from ContextValue we can use the parameter
-# "object_contextname" to query any object from the context which
-# is "dot-accessible". E.g we could use ObjectFieldValue("primary_email_address", "row")
-# or ObjectFieldValue("email", "row.primary_email_address")
-# in the second example we "extract the email-address object from the context and
-# use the field "email" of that object.
-#
-# The object can then be obtained from the context by calling super().resolve(context)
-
-
 class ObjectFieldLabel(hg.ContextValue):
-    def __init__(self, fieldname, object_contextname="object"):
+    def __init__(self, fieldname, object_contextname="object", title=True):
```

Comment:
Hmm the introduction of `object_contextname` seems to extend the functionality in a awkwardly limited way. Maybe we can just support dots in the fieldname so that arbitrary object names and nestings are possible...?

Reply:
Regarding the limitation: The name of the object can be dotted as well as long as the latest member of the dot-chain is a Django Model instance. And the fieldname can be dotted as well, but here we just try to be smart about that last members "verbose_name". So I think that is not a big concern.

The problem with a single dotted value would be that it would not be clear in all cases which part of the dot-chain represents the model instance. E.g. ```object.name``` it would be the first element, in ```request.user.email``` it would be the second element and in ```object.primary_adress.type.name``` it would be the third one.

But I agree that there is a bit of a problem here with the implicit assumption of ```object``` beeing in the context by default. This makes sense for Djangos class-based read and edit views, but not in other parts of the interface. A single string would make it a nicer API.

So, if we think having a single string (to declare the object and the label) is imporant there might be a solution, but it would not be trivial: We would need to resolve from "end-to-start" in order to determine the first member that is a Django model instance. But in order to go from end to start we need to look up pairwise because we cannot reverse-access a python object in general.
I think I can give it a short try, but if it doesn't work I will put in on lower priority, okay?

Reply:
I thought the fieldname never has dots. If it makes sense to allow dots in the fieldname, my suggestion is not relevant.